examples/notebooks/Creating Models/2-a-pde-model.ipynb

```diff
@@ -0,0 +1,317 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# initial conditions\n",
+    "model.initial_conditions = {c: pybamm.Scalar(1)}\n",
+    "\n",
+    "# boundary conditions\n",
+    "lbc = pybamm.Scalar(0)\n",
+    "rbc = pybamm.Scalar(2)\n",
+    "model.boundary_conditions = {c: {\"left\": (lbc, \"Dirichlet\"), \"right\": (rbc, \"Neumann\")}}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that in our example the boundary conditions take constant values, but the value can be any valid pybamm expression.\n",
+    "\n",
+    "Finally, we add any variables of interest to the dictionary `model.variables`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model.variables = {\"Concentration\": c, \"Flux\": N}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using the model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now the model is now completely defined all that remains is to discretise and solve. Since this model is a PDE we need to define the geometry on which it will be solved, and choose how to mesh the geometry and discretise in space."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Defining a geometry and mesh\n",
+    "\n",
+    "We can define spatial variables in a similar way to how we defined model variables, providing a domain and a coordinate system. The geometry on which we wish to solve the model is defined using a nested dictionary. The first key is the domain name (here \"negative particle\") and the entry is a dictionary giving the limits of the domain. Domains can have \"primary\", \"secondary\" or \"tabs\" dimensions. \"primary\" dimensions correspond to dimensions on which spatial operators will be applied (e.g. the gradient and divergence). In contrast, spatial operators do not act along \"secondary\" dimensions. This allows for multiple independent particles to be included into a model (such as the DFN model)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define geometry\n",
+    "r = pybamm.SpatialVariable(\n",
+    "    \"r\", domain=[\"negative particle\"], coord_sys=\"spherical polar\"\n",
+    ")\n",
+    "geometry = {\n",
+    "    \"negative particle\": {\n",
+    "        \"primary\": {r: {\"min\": pybamm.Scalar(0), \"max\": pybamm.Scalar(1)}}\n",
+    "    }\n",
+    "}"
```

Comment:
Not for this issue but it feels like there's one too many nested dictionaries in the geometry. I can't think of how to reformat to make it clearer while keeping the existing functionality, but maybe someone else can?